File: src/main/java/pl/funnyskaddon/core/commands/FunnySkAddonCommand.java
package pl.funnyskaddon.core.commands;

import org.bukkit.Bukkit;
import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;
import pl.funnyskaddon.core.FunnySkAddon;
import pl.funnyskaddon.core.utils.BasicUtil;

public class FunnySkAddonCommand implements CommandExecutor {
    @Override
    public boolean onCommand(CommandSender sender, Command cmd, String label, String[] args) {
        if(cmd.getName().equalsIgnoreCase("funnyskaddon")){
            if(sender.hasPermission("funnyskaddon.cmd")) {
                String java = System.getProperty("java.version");
                String engine = Bukkit.getServer().getBukkitVersion();
                String fsaVersion = FunnySkAddon.getInst().getDescription().getVersion();
                String fgVersion = Bukkit.getPluginManager().getPlugin("FunnyGuilds").getDescription().getVersion();
                String skriptVersion = Bukkit.getPluginManager().getPlugin("Skript").getDescription().getVersion();

                sender.sendMessage(BasicUtil.color("&8&m-----------------------------------------------------"));
                sender.sendMessage(BasicUtil.color("  &bInformacje o Dodatku"));
                sender.sendMessage(BasicUtil.color("   &7Wersja: &f" + FunnySkAddon.getInst().getDescription().getVersion()));
                sender.sendMessage(BasicUtil.color("   &7Autorzy: &f" + FunnySkAddon.getInst().getDescription().getAuthors()));
                sender.sendMessage(BasicUtil.color("   &7Opis: &f" + FunnySkAddon.getInst().getDescription().getDescription()));
                sender.sendMessage(BasicUtil.color("   &7Strona internetowa: &f" + FunnySkAddon.getInst().getDescription().getWebsite()));
                sender.sendMessage(BasicUtil.color("  &bWersje:"));
                sender.sendMessage(BasicUtil.color("   &7Java: &f" + java));
                sender.sendMessage(BasicUtil.color("   &7Bukkit: &f" + engine));
                sender.sendMessage(BasicUtil.color("   &7FunnySkAddon: &f" + fsaVersion));
                sender.sendMessage(BasicUtil.color("   &7FunnyGuilds: &f" + fgVersion));
                sender.sendMessage(BasicUtil.color("   &7Skript: &f" + skriptVersion));
                sender.sendMessage(BasicUtil.color("&8&m-----------------------------------------------------"));
            } else {
                sender.sendMessage("&bFunnySkAddon &8> &7Niestety nie posiadasz uprawnien.");
            }
        }
        return true;
    }
}

File: src/main/kotlin/pl/funnyskaddon/commands/FunnySkAddonCommand.kt
package pl.funnyskaddon.commands

import org.bukkit.Bukkit
import org.bukkit.command.Command
import org.bukkit.command.CommandExecutor
import org.bukkit.command.CommandSender
import pl.funnyskaddon.FunnySkAddon
import pl.funnyskaddon.util.color

class FunnySkAddonCommand(private val plugin: FunnySkAddon) : CommandExecutor {

    override fun onCommand(sender: CommandSender, cmd: Command?, label: String?, args: Array<String?>?): Boolean {
        if (!sender.hasPermission("funnyskaddon.cmd")) {
            sender.sendMessage(plugin.configuration.noPerm?.color())
            return true
        }

        val java = System.getProperty("java.version")
        val engine = Bukkit.getServer().bukkitVersion
        val fsaVersion = plugin.description.version
        val fgVersion = plugin.server.pluginManager.getPlugin("FunnyGuilds").description.version
        val skriptVersion = plugin.server.pluginManager.getPlugin("Skript").description.version

        sender.sendMessage("&8&m-----------------------------------------------------".color())
        sender.sendMessage("  &bInformacje o Dodatku".color())
        sender.sendMessage("   &7Wersja: &f".color() + plugin.description.version)
        sender.sendMessage("   &7Autorzy: &f".color() + plugin.description.authors)
        sender.sendMessage("   &7Opis: &f".color() + plugin.description.description)
        sender.sendMessage("   &7Strona internetowa: &f".color() + plugin.description.website)
        sender.sendMessage("  &bWersje:".color())
        sender.sendMessage("   &7Java: &f$java".color())
        sender.sendMessage("   &7Bukkit: &f$engine".color())
        sender.sendMessage("   &7FunnySkAddon: &f$fsaVersion".color())
        sender.sendMessage("   &7FunnyGuilds: &f$fgVersion".color())
        sender.sendMessage("   &7Skript: &f$skriptVersion".color())
        sender.sendMessage("&8&m-----------------------------------------------------".color())

        return true
    }

}